pkgs/development/python-modules/lockfile/default.nix

```diff
@@ -14,25 +14,28 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799";
+    hash = "sha256-au0C3gPLok76vNYAswVAFAY0/AbPpgOCLVCNU2Hp95k=";
   };
 
+  # The __init__ method doesn't do anything relevant as far as I can tell
+  # So we disable it, since it also makes pytest fail.
```

Comment:
I think the undoing of the monkey patch in the teardown function won’t work properly without it? But it could probably be moved to the class level, like the subclasses do.